internal/query/sessions_test.go

```diff
@@ -81,6 +80,7 @@ var (
 		"password_checked_at",
 		"intent_checked_at",
 		"passkey_checked_at",
```

Comment:
should this not be webauthn_checked_at?

Reply:
yes, you are right. Weird that the tests didn't fail. Appearantly you can put any arbirary string in the column names as long as the amount is correct...